app/javascript/packs/onboardingRedirectCheck.jsx

```diff
@@ -14,6 +14,8 @@ function redirectableLocation() {
     window.location.pathname !== '/onboarding' &&
     window.location.pathname !== '/signout_confirm' &&
     window.location.pathname !== '/privacy' &&
+    window.location.pathname !== '/code-of-conduct' &&
+    window.location.pathname !== '/terms' &&
```

Comment:
Adding these links here ensures that we do not attempt to redirect off of the Creator Settings form when the COC and TOS links are clicked.